src/index.js

```diff
@@ -8,7 +8,7 @@ class RealmAPI {
     if (!Authflow) throw new Error('Need to proive an Authflow instance to use the Realm API https://github.com/PrismarineJS/prismarine-auth')
     if (!PlatformTypes.includes(platform?.toLowerCase())) throw new Error(`Platform provided is not valid. Must be ${PlatformTypes.join(' | ')}`)
 
-    this.rest = new Rest(Authflow, platform, options)
+    this.rest = new Rest(Authflow, platform, { maxRetries: 5, ...options })
```

Comment:
This should only be done for specific call that is broken, it’s only the bedrock API join call that is flaky, no?

Reply:
Afaik yeah only the Bedrock join calls have issues so I'll change it to only retry that. Thought it might be a good thing to retry in case of server-side issues.